CONTRIBUTING.md

```diff
@@ -121,7 +121,37 @@ Many issues can be automatically fixed using the following command:
 npm run lint-fix
 ```
 
-## Submitting Pull Requests
+## Quality code
+
+Be confident in changing files you can check the quality (linter) with [pre-commit](https://pre-commit.com/index.html)
+Pre-commit is used to inspect the snapshot that is about to be committed, to see if you missed anything, or to review what you need to inspect in the code. You can see the actions descriptions in the **pre-commit-config.yml** file.
+
+The pre-commit is launched for PR validations, it is important to install the pre-commit locally to avoid unverified code being pushed.
+
+### Installation
+
+````
+pip install pre-commit
+ or
+$ brew install pre-commit (Homebrew)
+
+$ pre-commit install
+````
+
```

Comment:
```suggestion
````

Then run `pre-commit install` from the root of the openlibrary directory to set up the git hook scripts
```

Reply:
I don't consider this comment :)